pkg/template/controller/templateinstance_controller.go

```diff
@@ -436,7 +454,14 @@ func (c *TemplateInstanceController) instantiate(templateInstance *templateapi.T
 			ObjectTyper:  legacyscheme.Scheme,
 			ClientMapper: bulk.ClientMapperFromConfig(c.config),
 		},
+		DynamicMapper: &resource.Mapper{
+			RESTMapper:   c.dynamicRestMapper,
+			ObjectTyper:  legacyscheme.Scheme,
```

Comment:
A dynamic mapper that uses a legacy typer seems wrong.  What does it do with this and why wouldn't it be an unstructure typer of some kind?

Reply:
see above: i think the instructured typer is/was sending our legacy openshift objects to the k8s api endpoint instead of to oapi.

Reply:
The dynamic typer shouldn't be used for typed objects. I'd expect to be able to use an unstructured typer here.

Reply:
@liggitt @deads2k can one of you point me to the unstructured typer i should/could be using?

Reply:
> @liggitt @deads2k can one of you point me to the unstructured typer i should/could be using?

Does the `NewUnstructuredObjectTyper` not work?

Reply:
ok that seems to have worked, thanks.